alfasim_sdk/hook_specs.py

```diff
@@ -146,6 +146,18 @@ def update_plugins_secondary_variables(ctx: "void*") -> "int":
     """
 
 
+def update_plugins_secondary_variables_extra(ctx: "void*") -> "int":
```

Comment:
This hook's only used to update variables related with "tracers". Hence, I think that this hook should have a more meaningful name mentioning at least the tracers on his name.

Furthermore, we already have a hook named `update plugins secondary variables`, and adding a new hook just adding an `_extra` at the end could be confusing forcing the developer to explicitly look at the documentation to get the difference.

Reply:
But it is not only for tracers. The `ExtraVarSolver` can be used to solve extra variables, for instance, the moments from PBE of hydrates plugin.